features/step_definitions/breadcrumbs_steps.rb

```diff
@@ -0,0 +1,14 @@
+Given(/^the page exists$/) do
+  stub_content_store_has_item("/search", schema: "special_route")
```

Comment:
This should work for the search page, but you mentioned you wanted to use a different page to check it, so you can check one with a banner.

Which page would you like to use?

Reply:
EDIT: Or instead of doing all of that you can simply reuse an existing step, please, see a comment above.